1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,44 @@
 // Write your code here
+const items = document.querySelectorAll('img');
+const nextItem = document.querySelector('.next');
+const previousItem = document.querySelector('.previous');
+
+let count = 0;
+
+function showNextItem() {
+  items[count].classList.remove('active');
+  if(count < items.length - 1) {
+    count++;
+  } else {
+    count = 0;
+  }
+  items[count].classList.add('active');
+  }
+
+function showPreviousItem() {
+  items[count].classList.remove('active');
+  if(count > 0) {
+    count--;
+  } else {
+    count = items.length - 1;
+  }
+  items[count].classList.add('active');
+  }
+
+  function plusSlides(n) {
+    showNextItem(items[count] += n);
+    id = setTimeout(plusSlides, 1000)
+  }
+
+  function minesSlides(n) {
+    showPreviousItem(items[count] -= n);
+    id = setTimeout(minesSlides, 2000);
+  }
+  
+  function myStopFunction(){
+     clearTimeout(id);
+   }
```

Comment:
you have `clearTimeout(id);` here in the `myStopFunction`
but where does `id` come from?
i can only see it as local variables in other two functions above.
but because you declare `id` without `let`/`const`/`var` keyword, i assume browser might hoist it as global variable at the script's scope. however, this might have a great chance to end up as a bug in different browsers.

could you please correct it?